supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendLifecycleActorFactory.scala

```diff
@@ -41,6 +41,15 @@ case class JesBackendLifecycleActorFactory(configurationDescriptor: BackendConfi
     JesJobExecutionActor.props(jobDescriptor, jesConfiguration, initializationData.toJes.get, serviceRegistryActor).withDispatcher(BackendDispatcher)
   }
 
+  override def cacheHitCopyingActorProps(jobDescriptor: BackendJobDescriptor,
+                                         initializationData: Option[BackendInitializationData],
+                                         serviceRegistryActor: ActorRef,
+                                         cachedJobOutputs: JobOutputs): Props = {
+    // The `get` here is safe.  If the initialization data wasn't built, the workflow execution wouldn't have advanced to the
+    // point of trying to create job cache hit copying actors.
```

Comment:
I'll detail this a bit more in comments.  `JesInitializationActor#beforeAll` can only return an`Option(initializationData)` in a successful future, and we do have at least one test that checks the successful completion of the initialization future before marching onward in a workflow execution.